### PR TITLE
Added Klaytn Finder

### PR DESCRIPTION
**Pull request by praveen-klaytn**
## Proposed changes
In MORE section on the left panel, added below items
- Klaytn finder (Added new Environment variable KLAYTN_FINDER_URL)
- Renamed Klaytn IDE to Remix IDE

image

## Types of changes

Please put an x in the boxes related to your change.

- [X] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [X] I have read the [CONTRIBUTING](https://github.com/klaytn/klaytnwallet/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytnwallet)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by praveen-klaytn** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/components/Nav.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ if(KLAYTN_URL_NAME === 'Main Network'){
     { title: 'View Account Info', link: '/access', dropDown: false , icon: 'icon-info' },
     { title: 'Send KLAY & Token', link: '/transfer', search:'?next=transfer', dropDown: false ,  icon: 'icon-send' },
     { title: 'KLAY Faucet', link: '/faucet', dropDown: false ,search: '?next=faucet', icon: 'icon-faucet' },
-    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Klaytn IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
+    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Klaytnfinder',pageMove: true, subLink: KLAYTN_FINDER_URL}, {name: 'Remix IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
```
Comment (kjhman21) [inline src/components/Nav.js:25]:
```suggestion
    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnfinder',pageMove: true, subLink: KLAYTN_FINDER_URL}, {name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Remix IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
```

Comment (kjhman21) [reply src/components/Nav.js:25]: @praveen-klaytn can we make klaytnfinder before klaytnscope?

Comment (praveen-klaytn) [reply src/components/Nav.js]: Hi @kjhman21 , I have changed the order as requested.   
  
And I think, we need to make sure **KLAYTN\_FINDER\_URL** variable in added in **environment variables list of CI/CD** with [https://klaytnfinder.io/](https://www.klaytnfinder.io/) value for production and [https://baobab.klaytnfinder.io/](https://baobab.klaytnfinder.io/) value for baobab network. Default value of **KLAYTN\_FINDER\_URL** is  [https://klaytnfinder.io/](https://www.klaytnfinder.io/)

**Review by kjhman21 — APPROVED** [review]

File: src/components/Nav.js (hunk 1 above)
Comment (kjhman21) [reply src/components/Nav.js]: @praveen-klaytn Seems okay, but I will double-check after version update.

**Pull request merged**